api/v1alpha3/zz_generated.conversion.go

```diff
@@ -699,7 +694,6 @@ func autoConvert_v1alpha4_OpenStackClusterSpec_To_v1alpha3_OpenStackClusterSpec(
 	out.DNSNameservers = *(*[]string)(unsafe.Pointer(&in.DNSNameservers))
 	out.ExternalRouterIPs = *(*[]ExternalRouterIPParam)(unsafe.Pointer(&in.ExternalRouterIPs))
 	out.ExternalNetworkID = in.ExternalNetworkID
-	out.UseOctavia = in.UseOctavia
```

Comment:
UseOctavia should be false when converting v1alpha4 to v1alpha3.

Reply:
Ah I got what you mean. I though the other way around. It should be false, because boolean values in Go are per default false (this code is completely auto-generated and I cannot change anything manually).

Reply:
Sorry. yes, true.

Reply:
Thanks, I understand we can not set UseOctavia, since it has gone in v1alpha4.

Reply:
Yup:
* from v1alpha3 => v1alpha4 we cannot set it
* from v1alpha4 => v1alpha4 we don't set it, and thus it's false